src/util.c

```diff
@@ -37,6 +37,9 @@
 #define NI_BYTE_ARRAY_CHUNK	0xff	/* important: (2^n)-1 with all bits set */
 #define NI_BYTE_ARRAY_SIZE(len)	((len) | NI_BYTE_ARRAY_CHUNK)
 
+#define NI_BIT_SEPERATOR_FORMAT	"|"
```

Comment:
The parsing of the bitmask (names for values) and bitmap (names for bit numbers) types is the same, but they're differently formatted:
```
ni_dbus_deserialize_xml_scalar
        if (scalar_info->constraint.bitmask) 
                format: " | "
        if (scalar_info->constraint.bitmap)
                format: ", "
```
Perhaps we should  use different default separators for both:
```suggestion
#define NI_BITMAP_SEPERATOR_FORMAT	","
#define NI_BITMASK_SEPERATOR_FORMAT	"|"
```

Reply:
Ack I will change it!

Reply:
done